production/helm/loki/templates/single-binary/service.yaml

```diff
@@ -19,5 +19,5 @@ spec:
       targetPort: grpc
       protocol: TCP
   selector:
-    {{- include "loki.selectorLabels" . | nindent 4 }}
-{{- end }}
+    {{- include "loki.singleBinarySelectorLabels" . | nindent 4 }}
```

Comment:
@trevorwhitney unfortunately I don't know enough. This seems fine to me.

Reply:
Any approval for the PR then?